### PR TITLE
Raise alerts when node/cluster status changes

### PR DESCRIPTION
**Pull request by nnDarshan**
tendrl-bug-id: Tendrl/node-agent#714
Signed-off-by: Darshan N <darshan.n.2024@gmail.com>

### CODE REVIEW

File: tendrl/node_agent/node_sync/event_utils.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import json
+import os
+
+from tendrl.commons.utils import log_utils as logger
+from tendrl.commons.utils.time_utils import now as tendrl_now
+
+
+def emit_event(resource, curr_value, msg, instance,
```
Comment (shtripat) [inline tendrl/node_agent/node_sync/event_utils.py]: Should we move this function to commons and use in GI as well as node-agent?

Comment (nnDarshan) [reply tendrl/node_agent/node_sync/event_utils.py]: ack

File: tendrl/integrations/gluster/check_cluster_status.py [hunk 2]
```diff
@@ -26,6 +27,25 @@ def run():
                     "unhealthy",
                     prevExist=False
                 )
+
+                cluster_tendrl_context = NS.tendrl.objects.ClusterTendrlContext(
+                    integration_id=int_id
+                ).load()
+
+
+                msg = "Cluster {0} has went to unhealthy state".format(
```
Comment (shtripat) [inline tendrl/integrations/gluster/check_cluster_status.py]: Cluster <name> moved to unhealthy state?

Comment (nnDarshan) [reply tendrl/integrations/gluster/check_cluster_status.py]: ack

**Pull request merged**